rdr_service/api/participant_summary_api.py

```diff
@@ -53,8 +53,12 @@ def get(self, p_id=None):
                     raise Forbidden
             return super(ParticipantSummaryApi, self)._query("participantId")
 
-    def _make_query(self):
-        query = super(ParticipantSummaryApi, self)._make_query()
+    def _make_query(self, check_invalid=True):
+        pair_constraint, message = self._check_constraints()
+        if pair_constraint:
+            raise BadRequest(f"{message}")
```

Comment:
The ticket specifies that when HPRO searches we want to enforce that DOB and lastName be used together. I believe we still want to allow PTSC and the awardees to search by lastName or DOB without requiring both.

Reply:
Sure, was going on the ref to messages that were outlined in the doc and still edited it. Will change.

Reply:
Sure will add it now.